doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_topology.rst

```diff
@@ -0,0 +1,478 @@
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Signature verification policy    | signature_check_enabled_on_update_and_boot |                                                           |
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Resource Access Rights           |                                            | Does not operate on local resources                       |
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+
+System Controller Manifest
+==========================
+
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Property                         | Default Value                              | Note                                                      |
++==================================+============================================+===========================================================+
+| Vendor ID                        | 7617daa5-71fd-5a85-8f94-e28d735ce9f4       | RFC4122 uuid5(uuid.NAMESPACE_DNS, 'nordicsemi.com')       |
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Class ID                         | c08a25d7-35e6-592c-b7ad-43acc8d1d1c8       | RFC4122 uuid5(nordic_vid, 'nRF54H20_sys')                 |
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Downgrade prevention policy      | downgrade_prevention_enabled               |                                                           |
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Independent updateability policy | independent_update_denied                  |                                                           |
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Signature verification policy    | signature_check_enabled_on_update_and_boot |                                                           |
++----------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Resource Access Rights           |                                            | Ability to boot the System Controller,                    |
+|                                  |                                            | Memory access - TBD                                       |
```

Comment:
TBD?
Since this is not a manifest that a user can write/field that can configure, maybe it's best to simply skip it and write only about the booting abilities?